test/validation/api/timer/timer.c

```diff
@@ -408,7 +408,7 @@ static void timer_pool_create_destroy(void)
 	queue = odp_queue_create("timer_queue", &queue_param);
 	CU_ASSERT_FATAL(queue != ODP_QUEUE_INVALID);
 
-	memset(&tparam, 0, sizeof(odp_timer_pool_param_t));
+	odp_timer_pool_param_init(&tparam);
```

Comment:
The validation test changes could be split into a separate commit for clearer git history. It would also be good to add a simple test for checking `odp_timer_pool_param_t` default values.

Reply:
Both fixed in v2.